### PR TITLE
[Wallet] Account Key Cancel Confirmation

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

show a dialog when user goes to cancel prompting them to complete instead

### Tested

Screen Shot 2020-05-21 at 11 20 26 AM

### Related issues

- #3645 

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: packages/mobile/locales/en-US/backupKeyFlow6.json [hunk 1]
```diff
@@ -46,6 +46,10 @@
     "confirmation": "I have sent each Safeguard phrase to a trusted friend.",
     "yourSafeguards": "Your Safeguards"
   },
+  "cancelDialogTitle": "Finish setting up your Account Account Key",
```
Comment (jmrossy) [inline packages/mobile/locales/en-US/backupKeyFlow6.json:49]: Consider putting these under a shared property (e.g. `cancelDialog`) with 4 children. We have a bad habit of dumping everything at the top level but we shouldn't do that going forward, it's getting messy!

**Review by jmrossy — COMMENTED** [review]
Changes mostly lgtm, one comment about i18n string structure

File: packages/mobile/locales/en-US/backupKeyFlow6.json (hunk 1 above)
Comment (aaronmgdr) [reply packages/mobile/locales/en-US/backupKeyFlow6.json:49]: ha tots i usually do

**Review by i1skn — APPROVED** [review]
LGTM

**Pull request merged**